client.py

```diff
@@ -41,22 +42,99 @@ def send_encrypted_msg(self, message: str):
         encrypted_message = self.noise.encrypt(bytes(message, encoding="UTF-8"))
         self.sock.send(encrypted_message)
 
+    def send_messages(self):
+        print("You can write messages now, [q, quit, exit] to quit:")
+        while True:
+            user_input = input("Message: ")
+            if len(user_input) == 0:
+                print("Please input valid message, [q, quit, exit] to quit.")
+                continue
+            if user_input.lower() in ["q", "quit", "exit"]:
+                return
+            self.sock = socket.socket()
+            self.sock.connect((self.ip, self.port))
```

Comment:
Lines 55--59 seems to be identical with lines 76--80. Does it make sense to extract it to one method and work with that?

Reply:
Yes that could be done. I suspect that this will require rewrite if we want to use session for multiple messages. I will extract it and then create new branch and try multiple messages in one session.